.evergreen/run-tests.sh

```diff
@@ -221,7 +221,7 @@ fi
 
 if [ -n "$TEST_AUTH_OIDC" ]; then
     python -m pip install ".[aws]"
-    TEST_ARGS="test/auth_oidc/test_auth_oidc.py"
+    TEST_ARGS="test/auth_oidc/test_auth_oidc.py $TEST_ARGS"
```

Comment:
This is prepending `test/auth_oidc/test_auth_oidc.py` to the existing `TEST_ARGS`?

Reply:
Yes